anker/tests/tests/manager.rs

```diff
@@ -0,0 +1,125 @@
+use anker::error::AnkerError;
+use lido::{error::LidoError, token::Lamports};
+use solana_program::pubkey::Pubkey;
+use solana_program_test::tokio;
+use solana_sdk::{signature::Keypair, signer::Signer};
+use testlib::{
+    anker_context::{setup_token_pool, Context},
+    assert_solido_error,
+};
+
+const DEPOSIT_AMOUNT: Lamports = Lamports(1_000_000_000);
+
+#[tokio::test]
+async fn test_successful_change_token_swap_pool() {
+    let mut context = Context::new().await;
+    context
+        .initialize_token_pool_and_deposit(DEPOSIT_AMOUNT)
+        .await;
+    let mut new_token_pool = setup_token_pool(&mut context.solido_context).await;
+
+    // Copy UST token info from original Token Swap pool.
+    new_token_pool.ust_mint_address = context.token_pool_context.ust_mint_address;
+    let ust_mint_authority =
+        Keypair::from_bytes(&context.token_pool_context.ust_mint_authority.to_bytes()).unwrap();
+    new_token_pool.ust_mint_authority = ust_mint_authority;
+    new_token_pool.token_a = context
+        .solido_context
+        .create_spl_token_account(
+            new_token_pool.ust_mint_address,
+            new_token_pool.get_authority(),
+        )
+        .await;
+
+    new_token_pool
+        .initialize_token_pool(&mut context.solido_context)
+        .await;
+    let new_token_pool_address = new_token_pool.swap_account.pubkey();
+
+    let manager = Keypair::from_bytes(&context.solido_context.manager.to_bytes()).unwrap();
```

Comment:
Why jump through these hoops, we can just pass `&context.solido_context.manager` to `try_change_token_swap_pool` below?

Reply:
We can't, cause we borrow context as mutable before.

Reply:
Hmm, right. What we could do instead is to not take the manager as an argument in `try_change_token_swap_pool`, but use `self.solido_context.manager` directly inside that function. Then if we want to simulate an unauthorized call with the wrong manager, we still can, we just need to overwrite `context.solido_context.manager` with a new pubkey from the test. I think that will be a bit cleaner to read.